### ISSUE TITLE
a11y Tests Time Out

### ISSUE THREAD

**Issue by philippfromme** [bpmn-io/properties-panel#167]
__Describe the Bug__

Pull request checks often fail due to a11y tests timing out. These time-outs don't seem to be related to those pull requests.

```sh
Chrome Headless 92.0.4512.0 (Mac OS 10.15.7) <Simple> a11y should have no violations FAILED
	Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

__Expected Behavior__

a11y tests don't time out for no reason.

__Environment__

- Library version: 0.17.0
